packages/labs/analyzer/src/lib/javascript/variables.ts

```diff
@@ -32,16 +34,45 @@ type VariableName =
  * ts.Identifier within a potentially nested ts.VariableDeclaration.
  */
 const getVariableDeclaration = (
+  statement: ts.VariableStatement,
   dec: ts.VariableDeclaration | ts.EnumDeclaration,
```

Comment:
Ah, the EnumDeclaration in that union was vestigial; prior to this `getDeclarationInfo` handled both variables and enums, but after introducing the `statement` arg, I had to split them out. Removed.

Reply:
> I think statement can be retrieved from a VariableDeclaration with [.parent.parent](https://github.com/microsoft/TypeScript/blob/738b45e219c9a2febee72a55fdf532a7411a2c7b/src/compiler/types.ts#L1842-L1858).

Oh yeah good point. Hmm, it'd need to be cast though, since `VariableDeclaration` can exist in for/for of/for in statements also. Feels a little stricter as-is?

Reply:
What about checking if `.parent.parent` is a `VariableStatement`?

Reply:
What about checking it with `ts.isVariableStatement()`?